deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -76,7 +76,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: registry.k8s.io/sig-storage/csi-snapshotter:v5.0.1
+          image: registry.k8s.io/sig-storage/csi-snapshotter:v6.0.0
```

Comment:
the release notes at https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v6.0.0 point to k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.0

if it should be on registry.k8s.io, maybe report an issue in the github project?

Reply:
We replaced gcr with registry.k8s.io  here https://github.com/ceph/ceph-csi/pull/3047

Reply:
yes, I know. but if the release notes mention a different registry, I wonder how well the images are maintained elsewhere.